max.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "fmt"
+
+func max(x []string) string {
+	var l, s, p int
+	l = len(x)
+	s = len(x[0])
+	for i := 1; i < l; i++ {
+		if s < len(x[i]) {
```

Comment:
Please use `utf8.RuneCountInString()` function to get the length of utf-8 string.